backup/backupd.c

```diff
@@ -645,7 +645,7 @@ static void cmdloop(void)
             if (!backupd_userid) goto nologin;
             if (!strcmp(cmd.s, "Apply")) {
                 struct dlist *dl = NULL;
-                c = dlist_parse(&dl, /*parsekeys*/ 1, /*isbackup*/ 1, backupd_in);
+                c = dlist_parse(&dl, /*parsekeys*/ 1, /*isarchive*/ 0, /*isbackup*/ 1, backupd_in);
```

Comment:
I'd like to see us change the dlist functions to accept a single bitmask of flags rather than a function argument per flag.

Reply:
Yeah, that would be nice.  I've left a comment for next time we're in there.